docs/walkthrough.md

```diff
@@ -20,7 +20,7 @@ Detailed instructions can be found in the Kubernetes documentation under
 [Horizontal Pod
 Autoscaling](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-custom-metrics).
 
-Make sure that you've properly configured metrics-server (as is default in
+Make sure that you've properly configured the metrics-server (as default in
```

Comment:
metrics-server is the name of the specific thing, so it doesn't need a "the".

Reply:
you're right, sorry